packages/ui/src/components/breath-player/index.js

```diff
@@ -0,0 +1,89 @@
+import { Box } from '@mujo/box'
+import React, { useState, useMemo } from 'react'
+import { useTheme } from '../../hooks/use-theme'
+import { HeaderS } from '../fonts'
+import { createAnimationSteps } from './animation-steps'
+import { CenterText } from './center-text'
+import { Dots } from './dots'
+import { EndCardWrapper } from './end-card-wrapper'
+
+// const exampleJSON = [
```

Comment:
This is what the new animation JSON structure looks like. I might leave it this way for simplicity of debugging and then when we start to store/expose these animations to users I think it would be wise to compress it more to somethings like.

```json
[[[1.6,1],[1.8,1],[3000,3000]],[2.6,1],[2.8,1],[3200,3200]]
```